.github/workflows/ci.yml

```
@@ -16,116 +19,103 @@ jobs:
+            aws s3 cp s3://${S3_BUCKET}/backend/ . --recursive
+
+            # Install dependencies, build and start with PM2
+            if [ -f package.json ]; then
+              npm install || true
+              npm run build || true
+              pm2 stop my-app || true
+              pm2 start npm --name my-app -- start -- --port 8080
+              
+              # Configure nginx proxy
+              sudo bash -c 'cat > /etc/nginx/conf.d/app.conf << NGINXEOF
+server {
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 107: could not find expected ':'

(syntax-check)